civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -78,10 +133,13 @@ function _setup_modals() {
  * @param string $scriptFile the name of the script file (extension included)
  * @param string $page
  */
-function _add_script_to_page($scriptFile, $page) {
+function _add_script_to_page($scriptFile, $page = NULL) {
   if (arg()[0] == $page) {
     drupal_add_js(drupal_get_path('module', 'civihr_employee_portal') . "/js/$scriptFile");
   }
+  if (is_null($page)) {
```

Comment:
Maybe this would be better as
```php
// Exit if a page is specified and it's not the one we're currently on
if (!is_null($page) && arg()[0] != $page) {
  return;
}

// add script